x-pack/plugins/enterprise_search/server/lib/pipelines/create_pipeline_definitions.ts

```diff
@@ -269,26 +269,27 @@ export const formatMlPipelineBody = async (
       },
       {
         inference: {
-          model_id: modelId,
-          target_field: `ml.inference.${destinationField}`,
           field_map: {
             [sourceField]: modelInputField,
           },
+          model_id: modelId,
+          target_field: `ml.inference.${destinationField}`,
         },
       },
       {
         append: {
           field: '_source._ingest.processors',
           value: [
             {
-              type: modelType,
               model_id: modelId,
               model_version: modelVersion,
               processed_timestamp: '{{{ _ingest.timestamp }}}',
+              type: modelTypes,
```

Comment:
Should we rename this property to `types`, indicating that it's an array of potentially multiple type strings?

Reply:
As [mentioned in Slack](https://elastic.slack.com/archives/C02U50QNEAG/p1663874071207659?thread_ts=1663869133.233289&cid=C02U50QNEAG) - this is a good idea... incoming commit

Reply:
I don't think this is a schema we have control over, so we probably can't change this key.